dispatchers.go

```diff
@@ -313,6 +351,18 @@ func workdir(b *Builder, args []string, attributes map[string]bool, flagArgs []s
 	return nil
 }
 
+func parseCommandForHeredoc(firstline string) string {
+	// Expected format `RUN <str> <<`
+	if strings.HasPrefix(firstline, "RUN ") {
+		firstline = strings.TrimPrefix(firstline, "RUN ")
```

Comment:
This is case sensitive, while other parts of the parser are not.

Reply:
Fixed.